container/libcontainer/helpers.go

```diff
@@ -45,7 +45,7 @@ type CgroupSubsystems struct {
 // Get information about the cgroup subsystems.
 func GetCgroupSubsystems() (CgroupSubsystems, error) {
 	// Get all cgroup mounts.
-	allCgroups, err := cgroups.GetCgroupMounts()
+	allCgroups, err := cgroups.GetCgroupMounts(false)
```

Comment:
You eventually want to set this to true, right? It's fine to just do that in this PR (we frequently update dependencies while making changes that rely on them).